### ISSUE TITLE
apiserver-count is zero in HA setup.

### ISSUE THREAD

**Issue by hsyed** [kubernetes/kops#790]
I'm seeing a lot of these in /var/log/kube-apiserver.log on my masters. 3x HA.

```
W1102 22:46:05.395980       6 controller.go:351] Resetting endpoints for master service "kubernetes" to &TypeMeta{Kind:,APIVersion:,}
```

Googling indicates --apiserver-count should be set to 3 in this case but it is currently being set to 0.

**Comment by chrislovecnm** [comment]
Is this a functioning cluster out of the box?  What version of `kops`??

**Comment by hsyed** [comment]
The cluster is working beyond networking problems with weave net. Kops version is git-18cc365.

**Comment by justinsb** [comment]
Thanks for spotting!  This should be fixed now, in both 1.4.4 and master.  Please reopen if you continue to see this.
